src/app/containers/Timestamp/relativeTimestamp.js

```diff
@@ -0,0 +1,31 @@
+const relativeTime = receivedTimestamp => {
+  const currentTime = Date.now();
+  const timeDifferenceMilliseconds = currentTime - receivedTimestamp;
+  const toMinuteDivisor = 1000 * 60;
+  const timeDifferenceMinutes = parseInt(
+    timeDifferenceMilliseconds / toMinuteDivisor,
+    10,
+  );
+
+  if (timeDifferenceMinutes > 10 * 60 || timeDifferenceMilliseconds < 0) {
```

Comment:
```suggestion
  const isInFuture = timeDifferenceMilliseconds < 0;
  const isMoreThanTenHoursAgo = timeDifferenceMinutes > (10 * 60);
  if (isInFuture || isMoreThanTenHoursAgo) {
```

Here, though, would benefit from well-named one-off consts 😄